package.json

```diff
@@ -3,6 +3,9 @@
   "version": "1.0.0",
   "description": "Convert a given WSDL specification (1.1) to Postman Collection",
   "main": "index.js",
+  "bin": {
+    "openapi2postmanv2": "./bin/openapi2postmanv2.js"
```

Comment:
This should be `wsdl2postman` with the correct file name.

Reply:
Added in last commits